src/docdb/nodes.ts

```diff
@@ -0,0 +1,186 @@
+import * as vscode from 'vscode';
+import * as vm from 'vm';
+import * as path from 'path';
+import { EventEmitter, Event, Command } from 'vscode';
+import { AzureAccount } from '../azure-account.api';
+import { INode, ErrorNode } from '../nodes';
+import { ResourceManagementClient } from 'azure-arm-resource';
+import docDBModels = require("azure-arm-documentdb/lib/models");
+import DocumentdbManagementClient = require("azure-arm-documentdb");
+import { MongoDatabaseNode } from '../mongo/nodes';
+import {MongoCommands} from '../mongo/commands'
+var DocumentDBConnectionClient = require("documentdb").DocumentClient;
```

Comment:
I looked into this more and if you add `"@types/documentdb": "^1.10.2",` to devDependencies in package.json, you should be able to do the normal import like this:
```
import { DocumentClient } from 'documentdb';
```

Plus then you get all the helpful typings and stuff :)